### PR TITLE
Updated README and changes to install latest finder code.

### PR DESCRIPTION
**Pull request by ChandraAddala**

### CODE REVIEW

**Comment by ChandraAddala** [comment]
@usnavi @shintasmith @goru97 Pls review.

File: Dockerfile [hunk 1]
```diff
@@ -9,11 +9,7 @@ RUN apt-get update && \
     apt-get install -y libffi-dev --force-yes && \
     pip install gunicorn &&\
 	pip install --upgrade "git+http://github.com/rackerlabs/graphite-api.git@george/fetch_multi_with_patches" &&\
-	git -C /tmp clone https://github.com/rackerlabs/blueflood.git &&\
-    git -C /tmp/blueflood checkout master &&\
-	cd /tmp/blueflood/contrib/graphite &&\
-	python setup.py install
```
Comment (shintasmith) [inline Dockerfile:15]: What is this replaced with?

Comment (ChandraAddala) [inline Dockerfile:15]: `pip install --upgrade blueflood-graphite-finder`

Comment (shintasmith) [inline Dockerfile:15]: lol, too early in the morning for me to review, totally didn't see the line below.

**Comment by shintasmith** [comment]
lgtm

File: Dockerfile [hunk 2]
```diff
@@ -9,11 +9,7 @@ RUN apt-get update && \
     apt-get install -y libffi-dev --force-yes && \
     pip install gunicorn &&\
 	pip install --upgrade "git+http://github.com/rackerlabs/graphite-api.git@george/fetch_multi_with_patches" &&\
```
Comment (goru97) [inline Dockerfile:11]: Do we still need to rely on George's branch or are the changes merged with the master?

Comment (ChandraAddala) [inline Dockerfile:11]: They are not merged yet. @usnavi is working on it.

**Pull request merged**